src/components/Visualizer/Visualizer.component.jsx

```diff
@@ -72,8 +72,26 @@ class Visualizer extends React.Component {
         paddingWidth  - thisPaddingWidth -  
         borderWidth  -  thisBorderWidth -
         marginWidth - thisMarginWidth;
+
+        // //get padding, border and margin height for <div className={classes.visualizer}></div>
```

Comment:
Could you explain, what this does, and why did you comment it out.

Reply:
I have made a change and updated PR.

That part code which was comment out is to calculate height of canvas when window resize.

Regarding to visualizer is restarting when window resize. It is p5 canvas rendering , I have to look into p5 library to see whether it is possible to allow it to stop re-render when resize.